runtime/gear/src/constants.rs

```diff
@@ -63,4 +63,7 @@ pub mod time {
 
     // 1 in 4 blocks (on average, not counting collisions) will be primary BABE blocks.
     pub const PRIMARY_PROBABILITY: (u64, u64) = (1, 4);
+
+    // The free of charge period of rent.
+    pub const RENT_FREE_PERIOD: BlockNumber = pallet_gear_program::migration::FREE_PERIOD;
```

Comment:
why should we export it from migrations?

Reply:
The idea is that migrated programs should have the same free period as recently added programs. That explains the dependency.

Reply:
discussed